translate/cloud-client/translate_v3_list_glossary_test.py

```diff
@@ -34,7 +34,7 @@ def glossary():
     yield glossary_id
 
     try:
-        translate_v3_delete_glossary.sample_delete_glossary(PROJECT_ID, glossary_id)
+        translate_v3_delete_glossary.delete_glossary(PROJECT_ID, glossary_id)
     except Exception:
         pass
```

Comment:
Can we update this to not swallow exceptions? This would have been caught otherwise.

Reply:
Okay, I will make that change

Reply:
Did you mean for all of other samples inside translate v3?

Reply:
I mean short term for this specific sample, but if it's done in other samples they'll need to be adjusted sooner or later.

Reply:
I will adjust then all here.